packages/experimental/Menu/src/MenuItem/useMenuItem.ts

```diff
@@ -105,7 +105,7 @@ function getAccessibilityStateWorker(disabled: boolean, accessibilityState?: Acc
 }
 
 export const useHoverFocusEffect = (hovered: boolean, componentRef: React.MutableRefObject<any>) => {
-  React.useEffect(() => {
+  React.useLayoutEffect(() => {
```

Comment:
oof on the bug, but glad you caught it!

Reply:
lol thank EJ for making me change the colors

Reply:
Hover state on macOS matches rest (I only color the focus state differently), so I would have probably missed it too